view/frontend/templates/js/replacejs.phtml

```diff
@@ -1582,30 +1582,58 @@ $isLoadTrackJsDynamic = $block->isDisableTrackJsOnNonBoltPages() && $isLoadConne
     });
 
     if ((trim(location.pathname, '/') === 'checkout/cart') || (trim(location.pathname, '/') === 'checkout')) {
+        define("boltRequireOptional", [], {
+            load : function (moduleName, parentRequire, onload, config){
+                var onLoadSuccess = function(moduleInstance){
+                    // Module successfully loaded, call the onload callback so that
+                    // RequireJS can work its internal magic.
+                    onload(moduleInstance);
+                }
+                var onLoadFailure = function(err){
+                    // Module failed to load.
+                    if (err.requireModules) {
+                        var failedId = err.requireModules[0];
+                        console.warn("Could not load module: " + failedId);
```

Comment:
Was the console error the initial reason for this update? If so, please remove. Anyway, nicely done. Approving.

Reply:
@danac-gs No, usually it fails due to internal error, and we just log the general message.